src/main/java/fruitshop/model/dto/ReportDto.java

```diff
@@ -0,0 +1,18 @@
+package fruitshop.model.dto;
+
+import fruitshop.model.Fruit;
+import java.math.BigDecimal;
+import java.util.Map;
+import java.util.Set;
+
+public class ReportDto {
```

Comment:
this class looks redundant, you can just get entries from you dao and then do what you need with it

Reply:
done